src/workerd/server/workerd-api.c++

```diff
@@ -342,10 +343,53 @@ kj::Maybe<jsg::ModuleRegistry::ModuleInfo> WorkerdApi::tryCompileModule(
               module.getName(),
               module.getNodeJsCompatModule()));
     }
+    case config::Worker::Module::PYTHON_MODULE: {
+      // Nothing to do. Handled in compileModules.
+      return kj::none;
+    }
+    case config::Worker::Module::PYTHON_REQUIREMENT: {
+      // Nothing to do. Handled in compileModules.
+      return kj::none;
+    }
   }
   KJ_UNREACHABLE;
 }
 
+kj::String generatePyodideMetadata(config::Worker::Reader conf) {
+  kj::String result = kj::str("export function getMetadata() { return ");
+  capnp::JsonCodec jsonCodec;
+  jsonCodec.setPrettyPrint(false);
+  result = kj::str(result, jsonCodec.encode(conf));
+  result = kj::str(result, "; }");
+  return result;
+}
```

Comment:
Can the pyodide specific bits be moved out to a separate header/c++ file just to help keep things better organized.